database/common_models.py

```diff
@@ -53,6 +53,10 @@ def get_or_404(cls, *args, **kwargs):
     def query_set_as_unpacked_native_json(cls, query_set, remove_timestamps=True):
         return json.dumps([obj.as_unpacked_native_python(remove_timestamps) for obj in query_set])
 
+    @classmethod
+    def query_set_as_unpacked_native_python(cls, query_set, remove_timestamps=True):
+        return [obj.as_unpacked_native_python(remove_timestamps) for obj in query_set]
+
```

Comment:
Since `[obj.as_unpacked_native_python(remove_timestamps) for obj in query_set]` is in both this method and the above method, we should probably call `query_set_as_unpacked_native_python` from `query_set_as_unpacked_native_json` so as to not duplicate code.

Reply:
changed